edk2toolext/invocables/edk2_ci_build.py

```diff
@@ -132,8 +132,11 @@ def Go(self):
         pplist = self.PlatformSettings.GetPackagesPath() if self.PlatformSettings.GetPackagesPath() else []
 
         # Check Dependencies for Repo
+        # Generally we want the dependencies to be on the package path but not explictly
         for dependency in self.PlatformSettings.GetDependencies():
```

Comment:
why do you want to log a warning. ?

Reply:
I was thinking that if a dependency isn't on the packages path, it can cause rather strange errors. This was just to throw a warning to let a user know it's not on the path so it's a little easier to track down the error